druid/src/window.rs

```diff
@@ -182,8 +182,13 @@ impl<T: Data> Window<T> {
             );
         }
 
-        if self.root.state().needs_window_origin && !self.root.state().needs_layout {
-            let event = LifeCycle::Internal(InternalLifeCycle::ParentWindowOrigin);
+        if self.root.state().children_view_context_changed && !self.root.state().needs_layout {
```

Comment:
I think `self.root.state().children_view_context_changed` needs to be `widget_state.children_view_context_changed` instead.

When you resize the window `self.root.state().view_context_changed` changes to `true` but `self.root.state().children_view_context_changed` doesn't. The `widget_state` here is a temporary state acting as the virtual parent of the root, and thus has its `children_view_context_changed` as `true`.

For `self.root.state().needs_layout` it's less important and both should work the same, but for consistency I'd change that too to `widget_state.needs_layout`.

Reply:
Good catch :)